artifacts/rhcos/prerelease/rhcos.go

```diff
@@ -0,0 +1,81 @@
+package prerelease
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/containers/image/v5/pkg/compression/types"
+	"kubevirt.io/containerdisks/pkg/api"
+	"kubevirt.io/containerdisks/pkg/docs"
+	"kubevirt.io/containerdisks/pkg/hashsum"
+	"kubevirt.io/containerdisks/pkg/http"
+)
+
+type rhcos struct {
```

Comment:
It might be helpful to have type name consistent to the values from Metadata, maybe "prerelease"?

Reply:
I would use the package name here.

Reply:
I want to stick with `rhcos` on the struct. It is `rhcos` after all and keeps the method signatures shorter.